### ISSUE TITLE
Hideaways

### ISSUE THREAD

**Issue by TheChrisMorrison** [kjintroverted/clicky-button#3]
Hideaways = Hideouts
A place to store clicks so they can't be lost.
(More to come on how you lose clicks.)

**Comment by kjintroverted** [comment]
so this will get prioritized after we implement losing clicks..?

**Comment by TheChrisMorrison** [comment]
Yes, I imagine there will be rival clickers (hackers/writers) that can steal some of your clicks
